File: Twosum/s2.py
l=[]
n=int(input())
for i in range(0,n):
    ele=int(input())
    l.append(ele)
tar=int(input("enter tar"))
for i in range(0,n):
    for j in range(i+1,n):
        if(l[i]+l[j]==tar):
            print(i,j)

File: Twosum/s2.c

#include<stdio.h>
int main(void)
{
        int n,test,i,j;
        printf("enter the number of ele :");
        scanf("%d",&n);
        scanf("%d",&test);
        int a[n];
        for(i=0;i<n;i++){
                scanf("%d",&a[i]);
        }
        for(i=0;i<=n-1;i++){
                for(j=i+1;j<=n;j++){
                        if(a[i] + a[j] == test)
                         printf("%d %d",i,j);}}}
